shopping-cart-service/src/main/java/agh/fis/shopping_cart/service/ShoppingCartService.java

```diff
@@ -0,0 +1,65 @@
+package agh.fis.shopping_cart.service;
+
+import agh.fis.shopping_cart.model.*;
+import agh.fis.shopping_cart.client.ProductCatalogClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Optional;
+import java.util.*;
+import java.text.SimpleDateFormat;
+import java.math.BigDecimal;
+
+@Service
+public class ShoppingCartService {
+    private static final Logger logger = LoggerFactory.getLogger(ShoppingCartService.class);
+
+    private final ProductCatalogClient productCatalogClient;
+
+    public ShoppingCartService(ProductCatalogClient productCatalogClient) {
+        this.productCatalogClient = productCatalogClient;
+    }
+
+    public ShoppingCartDto create(ShoppingCartCreationDto shoppingCartCreationDto) {
+        Date date = new Date();
+        SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy HH:mm:ss");
+        ShoppingCartDto shoppingCartDto = createShoppingCartObject(10, shoppingCartCreationDto.getCustomerId(), formatter.format(date), new ArrayList<GameDto>());
+        return shoppingCartDto;
```

Comment:
Maybe we should already at this stage keep the customer shopping cart in DB? And then we will be able to create a simple implementation of `getShoppingCartById()`

Reply:
Also, I think we should create something like `getCustomerShoppingCart()` which would be run immediatly after customer is logged in.